flink-streaming-java/src/main/java/org/apache/flink/streaming/api/graph/StreamNode.java

```diff
@@ -402,7 +402,7 @@ public Optional<OperatorCoordinator.Provider> getCoordinatorProvider(
         }
     }
 
-    boolean isParallelismConfigured() {
+    public boolean isParallelismConfigured() {
```

Comment:
Why we need this change?

Reply:
Ah, At the beginning, I relied on it for the assertion of streamGraph's parallelism. Later, I changed the test logical, and really no longer needed it to be public now. The reason why not revert this is that I noticed that most of the other get methods on `StreamNode` are also public, and it will not change the internal state of the object. I prefer to keep this change for future convenience.

Reply:
It just looks confusing here because this change wasn't used in this pr, I tend to leave it alone, but it doesn't matter so it's up to you.

Reply:
Never mind. It's also reasonable to remove it, I have dropped the commit about this change and rebased all fix-up commits.